### PR TITLE
Update document for Azure LoadBalancer support.

### PR DESCRIPTION
**Pull request by karataliu**
Document update for following 2 PRs:
- Add support for bring-your-own ip address for Services on Azure: https://github.com/kubernetes/kubernetes/pull/42034
- Add support for Azure internal load balancer: https://github.com/kubernetes/kubernetes/pull/43510

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4082)

### CODE REVIEW

File: docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -429,10 +429,18 @@ with the user-specified `loadBalancerIP`. If the `loadBalancerIP` field is not s
 an ephemeral IP will be assigned to the loadBalancer. If the `loadBalancerIP` is specified, but the
 cloud provider does not support the feature, the field will be ignored.
 
-#### Internal load balancer on AWS
+Special notes for Azure: To use user-specified public type `loadBalancerIP`, a static type
+public IP address resource need to be created at first, and it should be in the same resource
```
Comment (chenopis) [inline docs/concepts/services-networking/service.md]: "need" -> "needs"

**Comment by karataliu** [comment]
Thanks for comment. Update pushed.

**Comment by chenopis** [comment]
ping @bprashanth

File: docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -429,10 +429,18 @@ with the user-specified `loadBalancerIP`. If the `loadBalancerIP` field is not s
 an ephemeral IP will be assigned to the loadBalancer. If the `loadBalancerIP` is specified, but the
 cloud provider does not support the feature, the field will be ignored.
 
-#### Internal load balancer on AWS
+Special notes for Azure: To use user-specified public type `loadBalancerIP`, a static type
+public IP address resource needs to be created at first, and it should be in the same resource
```
Comment (jdumars) [inline docs/concepts/services-networking/service.md]: public IP address resource needs to be created first, and it should be in the same resource

**Comment by jdumars** [comment]
Added one small suggestion.  Otherwise LGTM.

**Comment by chenopis** [comment]
@jdumars I'll merge this PR and make the adjustment in a followup PR.

**Pull request merged**